### PR TITLE
Add 'btn-small' to mod-stats-admin buttons

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue #22269 (kinda) .

### Summary of Changes
A minor tweak. Adds a `btn-small` class to the mod-stats-admin module buttons

### Testing Instructions
Check stats module in cpanel

### Before
image

### After
image

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on c2440f920d315ba1876ca498be86521db4611ddc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22276">issues.joomla.org/tracker/joomla-cms/22276</a>.</sub>

File: administrator/modules/mod_stats_admin/tmpl/default.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
 			</div>
 			<div class="span8">
 				<?php if(isset($item->link)) : ?>
-					<a class="btn btn-info js-revert" href ="<?php echo $item->link; ?>"><?php echo $item->data; ?></a>
+					<a class="btn btn-info btn-small js-revert" href ="<?php echo $item->link; ?>"><?php echo $item->data; ?></a>
```
Comment (Quy) [inline administrator/modules/mod_stats_admin/tmpl/default.php]: Please remove the space after `href`

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 579b2c6690bd247e83687acedfb44cce471948ae<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22276">issues.joomla.org/tracker/joomla-cms/22276</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22276">issues.joomla.org/tracker/joomla-cms/22276</a>.</sub>

**Pull request merged**